### DISCUSSION TITLE
[Docs] 打包文档能打包成中文吗？

### DISCUSSION THREAD

**Discussion by stdue** [element-plus/element-plus#8444] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `All`

## Description

没有看到能打包中文的选项，希望可以添加文档打包时打包成中文的选择，或者只是我没看见？如果有的话希望能告诉我

### Additional comments

_(empty)_

**Comment by sxzz** [comment]
目前非团队成员是无法下载 crowdin 中的翻译。我们后续可能会在 CI 中上传 crowdin 中的翻译内容。

Currently, The translated documents cannot be downloaded by non-team members. Perhaps we will upload the translated documents from crowdin in GitHub workflow.
